### ISSUE TITLE
Redirected list branches from old url adds 'x' in front of surname

### ISSUE THREAD

**Issue by BertKoor** [fisharebest/webtrees#5019]
Spotted an issue after my long overdue migration.
When an old url to a list of branches gets redirected, an 'x' is put at the front of the surname.

Analysis:
when getting the old url `/branches.php?ged=MyTree&surname=Doe`
it redirects to `/module/branches_list/Page/MyTree?soundex_dm=&soundex_std=&surname=Doe`
which in turn redirects to `/tree/MyTree/branches/xDoe?soundex_dm=0&soundex_std=0`

I found the issue is in function getPageAction of `/app/Module/BranchesListModule.php` :

````
'surname'     => 'x' . Validator::queryParams($request)->string('surname'), 
````

I'm preparing a PR to fix this myself

**Comment by BertKoor** [comment]
Here's the PR with a fix: https://github.com/fisharebest/webtrees/pull/5020

**Comment by fisharebest** [comment]
Closed by #5020
